operational_graphics/toolboxes/scripts/NumberFeatures.py

```diff
@@ -89,6 +91,22 @@ def findLayerByName(layerName):
     else:
         arcpy.AddMessage("Non-map environment, no layers to find...")
 
+
+def GetApplication():
```

Comment:
I thought this was moved to a "utility" script - but this may be a work in progess/TODO